pynecone/compiler/templates.py

```diff
@@ -35,22 +35,21 @@ def format_import(lib: str, default: str = "", rest: Optional[Set[str]] = None)
         The compiled import statement.
     """
     # Handle the case of direct imports with no libraries.
-    if lib == "":
-        assert default == "", "No default field allowed for empty library."
+    if not lib:
+        assert not default, "No default field allowed for empty library."
         assert rest is not None and len(rest) > 0, "No fields to import."
         return join([IMPORT_LIB(lib=lib) for lib in sorted(rest)])
 
     # Handle importing from a library.
     rest = rest or set()
-    if len(default) == 0 and len(rest) == 0:
+    if not default and len(rest) == 0:
```

Comment:
Also can we keep the syntax
```
len(...) == 0
```
everywhere rather than `not ...` . It is equivalent but the former is easier to reason about semantically.

I think it's largely a matter of opinion, but I'd rather not have the implicit bool casting Python does in this codebase to keep things clear.